### PR TITLE
feat(editor,api,db,example): HTML Block

### PR DESCRIPTION
**Pull request by antkiewiczk**

### CODE REVIEW

File: packages/editor/src/client/locales/fr.json [hunk 1]
```diff
@@ -788,7 +796,7 @@
       "create": "Créer",
       "save": "Sauvegarder",
       "password": "Mot de passe",
-      "paidUntil": "Payé jusqu'à",
+      "payedUntil": "Payé jusqu'à",
```
Comment (Itrulia) [inline packages/editor/src/client/locales/fr.json:799]: Merge error

File: packages/editor/src/client/locales/de.json [hunk 2]
```diff
@@ -925,7 +933,7 @@
       "name": "Name",
       "password": "Passwort",
       "passwordChangeSuccess": "Passwort erfolgreich geändert!",
-      "paidUntil": "Bezahlt bis",
+      "payedUntil": "Bezahlt bis",
```
Comment (Itrulia) [inline packages/editor/src/client/locales/de.json:936]: Merge error

File: packages/editor/src/client/panel/htmlEditPanel.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+import {HTMLBlockValue} from '../blocks/types'
+
+export interface HtmlEditPanelProps {
+  readonly value: HTMLBlockValue
+  onClose(): void
+  onConfirm(value: HTMLBlockValue): void
+}
+
+export function HtmlEditPanel({value, onClose, onConfirm}: HtmlEditPanelProps) {
+  const [input, setInput] = useState<HTMLBlockValue>(value)
```
Comment (Itrulia) [inline packages/editor/src/client/panel/htmlEditPanel.tsx:14]: We don't need input just `htmlBlock`

File: packages/editor/src/client/panel/htmlEditPanel.tsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  const [htmlBlock, setHtmlBlock] = useState<HTMLBlockValue>(value)
+  const isEmpty = htmlBlock === undefined
+  const {t} = useTranslation()
+
+  useEffect(() => {
```
Comment (Itrulia) [inline packages/editor/src/client/panel/htmlEditPanel.tsx:19]: This is not needed when input is being replaced

**Review by Itrulia — CHANGES_REQUESTED** [review]

**Review by Itrulia — APPROVED** [review]

File: packages/editor/src/client/blocks/htmlBlock.tsx [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+                  right: 0
+                }}>
+                <IconButton
+                  size={'lg'}
+                  icon={<PencilIcon />}
+                  onClick={() => setHtmlDialogOpen(true)}>
+                  {t('blocks.html.edit')}
+                </IconButton>
+              </div>
+              <div style={{marginTop: '30px'}} dangerouslySetInnerHTML={{__html: value.html}} />
```
Comment (tomaszdurka) [inline packages/editor/src/client/blocks/htmlBlock.tsx:54]: What do you think about not showing it in editor? :)

**Review by tomaszdurka — APPROVED** [review]
Looks good, a little bit of security issues with setting especially in editor. But let's say it's good :)

**Review by tomaszdurka — APPROVED** [review]

**Review by Itrulia — APPROVED** [review]

**Pull request merged**